subworkflows/nf-core/preprocess_rnaseq/tests/main.nf.test

```diff
@@ -18,24 +18,31 @@ nextflow_workflow {
     tag "subworkflows/fastq_fastqc_umitools_fastp"
     tag "subworkflows/fastq_subsample_fq_salmon"
 
+
+
     test("homo_sapiens paired-end [fastq] fastp") {
 
         when {
             workflow {
                 """
-                input[0] = Channel.of([
+                ch_reads = Channel.of([
                     [ id:'test', single_end:false, strandedness:'auto' ], // meta map
                     [
                         file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/fastq/test_rnaseq_1.fastq.gz', checkIfExists: true),
                         file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/fastq/test_rnaseq_2.fastq.gz', checkIfExists: true)
                     ]
-                ]) // ch_reads
+                ])
+
+                ch_ribo_db = file('ribo_db.txt')
+                ch_ribo_db.append('https://raw.githubusercontent.com/biocore/sortmerna/v4.3.4/data/rRNA_databases/rfam-5.8s-database-id98.fasta')
```

Comment:
this rRNA db not in nf-core test-datasets repo ?

Reply:
I could copy it in, but didn't seem necessary. This is one of the links used again and again by rnaseq, so it's stable